app/src/main/java/com/sun/spoonacular/ui/home/HomeFragment.kt

```diff
@@ -16,7 +31,60 @@ class HomeFragment : Fragment() {
         return inflater.inflate(R.layout.fragment_home, container, false)
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initDataHome()
+    }
+
+    private fun initDataHome() {
+        recipes =
+            arguments?.getParcelableArrayList<Recipe>(BUNDLE_LIST_RECIPE_KEY) as ArrayList<Recipe>
+        for (i in 0..4) {
+            recipeSlide.add(recipes[i])
+        }
```

Comment:
```suggestion
        for (item in recipes) {
            recipeSlide.add(item)
        }
```

Reply:
Chỗ này em không lấy hết cái list a ơi, em chỉ lấy 5 phần tử đầu thôi, còn lại là để gán data cho recyler view bên dưới

Reply:
vậy thì sửa for loop lại. trường hợp data trả về ít hơn 5 phần tử thì for sao chạy đến 4 được e